README.md

```diff
@@ -47,7 +47,13 @@ Recommended plugins to have are:
 $ npm install -g yarn@0.21.3
 ```
 
-3) Now install the build and application dependencies by running `$ yarn install` (Vagrant will do this for you)
+3) Now install the build and application dependencies by running 
+
+```
+$ yarn install
+```
+
+(Vagrant will do this for you)
```

Comment:
This line could actually be deleted
>(Vagrant will do this for you)

since there is no Vagrant support in this project.  (copy / 🍝  strike again!)